lib/stdlib/src/proplists.erl

```diff
@@ -39,6 +39,7 @@
 %% etc.</p>
 %%
 %% % @type property() = atom() | tuple()
+%% % @type proplist() = [property()]
```

Comment:
I am 100% with @fenollp on this one.  Edoc should really be using the `export` and `export_type` (and from there the `spec` and `type` and `opaque`) declarations of the module, not force the programmer to repeat them in comments -- with the possibility to introduce errors and other inconsistencies in the process.